node/src/main/scala/org/bitcoins/node/models/PeerDAO.scala

```diff
@@ -0,0 +1,45 @@
+package org.bitcoins.node.models
+
+import org.bitcoins.db.{CRUD, SlickUtil}
+import org.bitcoins.node.config.NodeAppConfig
+import slick.lifted.ProvenShape
+
+import java.time.Instant
+import scala.concurrent.{ExecutionContext, Future}
+
+case class PeerDB(
+    address: String,
+    lastConnected: Instant
```

Comment:
maybe add `firstSeen`?

Reply:
may want to add `networkId` for differentiating tor vs clearnet vs i2p, etc

https://github.com/bitcoin-s/bitcoin-s/blob/1c6d728197e7276e319b912a184311ae493b02fe/core/src/main/scala/org/bitcoins/core/p2p/NetworkPayload.scala#L506